src/api/app/controllers/request_controller.rb

```diff
@@ -152,8 +152,12 @@ def request_create
     end
 
     # cache the diff (in the backend)
-    @req.bs_request_actions.each do |a|
-      BsRequestActionWebuiInfosJob.perform_later(a)
+    @req.bs_request_actions.each do |action|
+      # cleanup implicit home branches.
+      # FIXME3.0: remove this, the clients should do this automatically meanwhile
```

Comment:
What does `3.0` refer to? OBS release?

Reply:
Yeah - you will notice I just moved the comment though

Reply:
I did notice, but it didn't answer my question knowing this. Now I know, thank you.